server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -523,38 +644,51 @@ private TsFileProcessor createTsFileProcessor(boolean sequence)
       baseDir = DirectoryManager.getInstance().getNextFolderForUnSequenceFile();
     }
     fsFactory.getFile(baseDir, storageGroupName).mkdirs();
-    String filePath = baseDir + File.separator + storageGroupName + File.separator +
-        System.currentTimeMillis() + IoTDBConstant.TSFILE_NAME_SEPARATOR + versionController
-        .nextVersion() + IoTDBConstant.TSFILE_NAME_SEPARATOR + "0" + TSFILE_SUFFIX;
 
+    String filePath =
+        baseDir + File.separator + storageGroupName + File.separator + timeRangeId + File.separator
```

Comment:
I wonder if this timeRangeId should be replaced by Date, which is meaningful to users.

Reply:
If user change interval to week or hour, this can't be called Date